Makefile

```diff
@@ -0,0 +1,62 @@
+# Variables to compile go client
+GO_CLIENT=bitwrk-client
+GO_RELEASE_DIST=./dist
+CLIENT_LINUX=$(GO_RELEASE_DIST)/bitwrk_linux_amd64/$(GO_CLIENT)
+CLIENT_DARWIN=$(GO_RELEASE_DIST)/bitwrk_darwin_amd64/$(GO_CLIENT)
+CLIENT_WINDOWS=$(GO_RELEASE_DIST)/bitwrk_windows_amd64/$(GO_CLIENT).exe
+ADDON_NAME_ROOT=render_bitwrk
+VERSION=0.7.0
+
+# Variables to zip addon and client daemon into one 
+TMPDIR=tmp
```

Comment:
Makefile fails when tmp is not there, i.e. always initially.
I don't really understand the error message:
```
$ make
goreleaser --snapshot --skip-publish --rm-dist

   • releasing using goreleaser dev...
   • loading config file       file=.goreleaser.yml
   • RUNNING BEFORE HOOKS
      • running go mod download
   ⨯ release failed after 0.03s error=hook failed: go mod download
go: creating work dir: stat tmp: no such file or directory
```
Running `mkdir tmp` first fixes the problem but shouldn't be necessary.

Reply:
I don't see this problem when testing it in Linux. It seems like 'mkdir' is failing when running from within the Makefile. Could it be a problem with permissions?

Reply:
Do you have Go installed and its bin directory in the PATH environment variable? Are you running the makefile in Mac, Linux, Windows? This problem could also be due to the version of Go being too old (therefore not supporting modules).